packages/block-library/src/categories/edit.js

```diff
@@ -1,11 +1,16 @@
+/**
```

Comment:
I noticed that `loadash` was located in the wrong section of imports. It's fixed now.

Reply:
Thanks! I wasn't sure what order to put it in I'll make sure to check out the documentation for the JS standards